bukuserver/templates/bukuserver/home.html

```diff
@@ -4,14 +4,16 @@
 {% block head %}
   {{ super() }}
   {{ buku.close_if_popup() }}
-  {{ buku.focus('form[action="/"]') }}
+  {{ buku.focus('main form[action="/"]') }}
 {% endblock %}
 
 {% block menu_links %}
 {{ super() }}
-<form class="navbar-form navbar-right" action="{{ url_for('bookmark.index_view') }}" method="GET">
+<form class="navbar-form navbar-right" action="{{ url_for('admin.search') }}" method="POST">
```

Comment:
Search splitting is done in the `admin.search` handler, thus replacing the `action=` value.